dot-net-sdk/store/ConfigurationStore.cs

```diff
@@ -8,83 +8,48 @@ namespace eppo_sdk.store;
 
 public class ConfigurationStore : IConfigurationStore
 {
-    private readonly MemoryCache _experimentConfigurationCache;
+    private readonly MemoryCache _flagConfigurationCache;
     private readonly MemoryCache _banditModelCache;
     private readonly ConfigurationRequester _requester;
-    private static ConfigurationStore? _instance;
 
-    public ConfigurationStore(ConfigurationRequester requester, MemoryCache flagConfigurationCache, MemoryCache banditModelCache)
+    public ConfigurationStore(ConfigurationRequester requester,
+                              MemoryCache flagConfigurationCache,
+                              MemoryCache banditModelCache)
     {
         _requester = requester;
-        _experimentConfigurationCache = flagConfigurationCache;
+        _flagConfigurationCache = flagConfigurationCache;
         _banditModelCache = banditModelCache;
     }
 
-    public static ConfigurationStore GetInstance(MemoryCache flagConfigurationCache, MemoryCache banditModelCache,
```

Comment:
I spent the better part of the day trying to track down why `BanditClientTest` and `EppoClientTest` could each run successfully on their own, but not together. .NET uses the same execution memory for all test cases/fixtures. While `EppoClient` allows for multiple instantiations, this unforgetting singleton pattern here caused configuration from `BanditClientTest` to leak into and break `EppoClientTest` which only made the mistake of coming later in the alphabet.
Enforcing a singleton here doesn't add anything at a higher level and doesn't make sense given it can be called multiple times with different arguments. I explored indexing the instances by URL, but it only added complexity.